### PR TITLE
Deploy pyusd

### PR DESCRIPTION
**Pull request by pmckelvy1**

### CODE REVIEW

**Comment by pmckelvy1** [comment]
https://etherscan.io/address/0x7622d205b88e5fb7030e5d717ade7202592b31ab#code

File: hardhat.config.ts [hunk 1]
```diff
@@ -81,9 +83,7 @@ const config: HardhatUserConfig = {
     mainnet: {
       chainId: 1,
       url: MAINNET_RPC_URL,
-      accounts: {
-        mnemonic: MNEMONIC,
-      },
+      accounts: [DEPLOYER_PK],
```
Comment (akshatmittal) [inline hardhat.config.ts:86]: Do not use PK, use the mnemonic setup instead. Several scripts in the project depend on having multiple available accounts, and generally speaking it's better anyway.

File: scripts/verification/assets/verify_pyusd.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  if (developmentChains.includes(hre.network.name)) {
+    throw new Error(`Cannot verify contracts for development chain ${hre.network.name}`)
+  }
+
+  deployments = <IAssetCollDeployments>getDeploymentFile(getAssetCollDeploymentFilename(chainId))
+
+  const asset = await hre.ethers.getContractAt('Asset', deployments.assets.pyUSD!)
+
+  /** ******************** Verify RSR Asset ****************************************/
```
Comment (tbrent) [inline scripts/verification/assets/verify_pyusd.ts:26]: comment

File: hardhat.config.ts (hunk 1 above)
Comment (tbrent) [inline hardhat.config.ts:86]: why add this DEPLOYER_PK approach? trying to separate out keys used for different purposes?

Comment (tbrent) [reply hardhat.config.ts:86]: maybe you didn't mean to check this in?

**Comment by tbrent** [comment]
deployed address LGTM 👍

File: hardhat.config.ts (hunk 1 above)
Comment (pmckelvy1) [reply hardhat.config.ts:86]: yeah i generally prefer to use a pk, as it is less surface area for moving sensitive info around. if i accidentally check in the seed then that entire wallet is compromised (not that i'm putting anything significant on these addresses anyways.
i'll roll it back

File: scripts/deploy.ts [hunk 3]
```diff
@@ -88,7 +88,8 @@ async function main() {
       'phase2-assets/collaterals/deploy_apxeth.ts',
       'phase2-assets/collaterals/deploy_USDe.ts',
       'phase2-assets/assets/deploy_crv.ts',
-      'phase2-assets/assets/deploy_cvx.ts'
+      'phase2-assets/assets/deploy_cvx.ts',
+      'phase2-assets/assets/deploy_pyUSD.ts'
```
Comment (tbrent) [inline scripts/deploy.ts:92]: case mismatch with file name

**Review by tbrent — APPROVED** [review]
nit: update `scripts/verify_etherscan.ts` too, LGTM after

**Review by tbrent — APPROVED** [review]

**Review by akshatmittal — APPROVED** [review]

**Pull request merged**